### ISSUE TITLE
Enupal installation fails with db exception.

### ISSUE THREAD

**Issue by Bergrebell** [enupal/translate#17]
### Description
Enupal installation fails with db exception.

### Steps to reproduce

1. require via UI or through `composer require `enupal/translate`
2. install via /admin/settings/plugins

### Additional info

- Craft version: 3.0.36
- PHP version: 7.2.11
- Database driver & version: Maria DB
- Plugin version: 1.1

### Stacktrace
```
PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'settings' in 'field list' in /projectname/.../vendor/yiisoft/yii2/db/Command.php:1258
Stack trace:
#0 /projectname/.../vendor/yiisoft/yii2/db/Command.php(1258): PDOStatement->execute()
#1 /projectname/.../vendor/yiisoft/yii2/db/Command.php(1148): yii\db\Command->internalExecute('SELECT `setting...')
#2 /projectname/.../vendor/yiisoft/yii2/db/Command.php(413): yii\db\Command->queryInternal('fetch', NULL)
#3 /projectname/.../vendor/yiisoft/yii2/db/Query.php(274): yii\db\Command->queryOne()
#4 /projectname/.../vendor/craftcms/cms/src/db/Query.php(177): yii\db\Query->one(NULL)
#5 /projectname/.../vendor/enupal/translate/src/services/Settings.php(63): craft\db\Query->one()
#6 /projectname/.../vendor/enupal/translate/src/Translate.php(40): enupal\translate\services\Settings->getDbSettings()
#7 /projectname/.../vendor/yiisoft/yii2/base/BaseObject.php(109): enupal\translate\Translate->init()
#8 /projectname/.../vendor/yiisoft/yii2/base/Module.php(158): yii\base\BaseObject->__construct(Array)
#9 /projectname/.../vendor/craftcms/cms/src/base/Plugin.php(127): yii\base\Module->__construct('enupal-translat...', Object(craft\web\Application), Array)
#10 [internal function]: craft\base\Plugin->__construct('enupal-translat...', Object(craft\web\Application), Array)
#11 /projectname/.../vendor/yiisoft/yii2/di/Container.php(383): ReflectionClass->newInstanceArgs(Array)
#12 /projectname/.../vendor/yiisoft/yii2/di/Container.php(156): yii\di\Container->build('enupal\\translat...', Array, Array)
#13 /projectname/.../vendor/yiisoft/yii2/BaseYii.php(349): yii\di\Container->get('enupal\\translat...', Array, Array)
#14 /projectname/.../vendor/craftcms/cms/src/services/Plugins.php(847): yii\BaseYii::createObject(Array, Array)
#15 /projectname/.../vendor/craftcms/cms/src/services/Plugins.php(468): craft\services\Plugins->createPlugin('enupal-translat...')
#16 /projectname/.../vendor/craftcms/cms/src/controllers/PluginsController.php(51): craft\services\Plugins->installPlugin('enupal-translat...', NULL)
#17 [internal function]: craft\controllers\PluginsController->actionInstallPlugin()
#18 /projectname/.../vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#19 /projectname/.../vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#20 /projectname/.../vendor/craftcms/cms/src/web/Controller.php(109): yii\base\Controller->runAction('install-plugin', Array)
#21 /projectname/.../vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('install-plugin', Array)
#22 /projectname/.../vendor/craftcms/cms/src/web/Application.php(294): yii\base\Module->runAction('plugins/install...', Array)
#23 /projectname/.../vendor/craftcms/cms/src/web/Application.php(554): craft\web\Application->runAction('plugins/install...', Array)
#24 /projectname/.../vendor/craftcms/cms/src/web/Application.php(278): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#25 /projectname/.../vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#26 /projectname/.../public/index.php(24): yii\base\Application->run()
#27 {main}

Next yii\db\Exception: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'settings' in 'field list'
The SQL being executed was: SELECT `settings`
FROM `plugins`
WHERE `handle`='enupal-translate'
LIMIT 1 in /projectname/.../vendor/yiisoft/yii2/db/Schema.php:664
Stack trace:
#0 /projectname/.../vendor/yiisoft/yii2/db/Command.php(1263): yii\db\Schema->convertException(Object(PDOException), 'SELECT `setting...')
#1 /projectname/.../vendor/yiisoft/yii2/db/Command.php(1148): yii\db\Command->internalExecute('SELECT `setting...')
#2 /projectname/.../vendor/yiisoft/yii2/db/Command.php(413): yii\db\Command->queryInternal('fetch', NULL)
#3 /projectname/.../vendor/yiisoft/yii2/db/Query.php(274): yii\db\Command->queryOne()
#4 /projectname/.../vendor/craftcms/cms/src/db/Query.php(177): yii\db\Query->one(NULL)
#5 /projectname/.../vendor/enupal/translate/src/services/Settings.php(63): craft\db\Query->one()
#6 /projectname/.../vendor/enupal/translate/src/Translate.php(40): enupal\translate\services\Settings->getDbSettings()
#7 /projectname/.../vendor/yiisoft/yii2/base/BaseObject.php(109): enupal\translate\Translate->init()
#8 /projectname/.../vendor/yiisoft/yii2/base/Module.php(158): yii\base\BaseObject->__construct(Array)
#9 /projectname/.../vendor/craftcms/cms/src/base/Plugin.php(127): yii\base\Module->__construct('enupal-translat...', Object(craft\web\Application), Array)
#10 [internal function]: craft\base\Plugin->__construct('enupal-translat...', Object(craft\web\Application), Array)
#11 /projectname/.../vendor/yiisoft/yii2/di/Container.php(383): ReflectionClass->newInstanceArgs(Array)
#12 /projectname/.../vendor/yiisoft/yii2/di/Container.php(156): yii\di\Container->build('enupal\\translat...', Array, Array)
#13 /projectname/.../vendor/yiisoft/yii2/BaseYii.php(349): yii\di\Container->get('enupal\\translat...', Array, Array)
#14 /projectname/.../vendor/craftcms/cms/src/services/Plugins.php(847): yii\BaseYii::createObject(Array, Array)
#15 /projectname/.../vendor/craftcms/cms/src/services/Plugins.php(468): craft\services\Plugins->createPlugin('enupal-translat...')
#16 /projectname/.../vendor/craftcms/cms/src/controllers/PluginsController.php(51): craft\services\Plugins->installPlugin('enupal-translat...', NULL)
#17 [internal function]: craft\controllers\PluginsController->actionInstallPlugin()
#18 /projectname/.../vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#19 /projectname/.../vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#20 /projectname/.../vendor/craftcms/cms/src/web/Controller.php(109): yii\base\Controller->runAction('install-plugin', Array)
#21 /projectname/.../vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('install-plugin', Array)
#22 /projectname/.../vendor/craftcms/cms/src/web/Application.php(294): yii\base\Module->runAction('plugins/install...', Array)
#23 /projectname/.../vendor/craftcms/cms/src/web/Application.php(554): craft\web\Application->runAction('plugins/install...', Array)
#24 /projectname/.../vendor/craftcms/cms/src/web/Application.php(278): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#25 /projectname/.../vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#26 /projectname/.../public/index.php(24): yii\base\Application->run()
#27 {main}
Additional Information:
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'settings' in 'field list'
)
```

**Comment by andrelopez** [comment]
Thanks @Bergrebell I'm on it

**Comment by andrelopez** [comment]
Hi @Bergrebell  this issue is fixed in Enupal Translate v1.1.8

Happy holidays!
